### ISSUE TITLE
gif images are not animated

### ISSUE THREAD

**Issue by a1ekseev** [wallabag/wallabag#6317]

**Comment by j0k3r** [comment]
I just tried on wallabag.it and the gif are animated
<img width="1147" alt="image" src="https://user-images.githubusercontent.com/62333/220049824-6129da3f-0f82-40b8-9d3a-013e7bb46b54.png">

If you want gif support you need to install the extension `php-imagick` as stated in the doc: https://doc.wallabag.org/en/admin/internal_settings.html#download-images-locally

Are you using the official docker image?

**Comment by a1ekseev** [comment]
> I just tried on wallabag.it and the gif are animated <img alt="image" width="1147" src="https://user-images.githubusercontent.com/62333/220049824-6129da3f-0f82-40b8-9d3a-013e7bb46b54.png">
> 
> If you want gif support you need to install the extension `php-imagick` as stated in the doc: https://doc.wallabag.org/en/admin/internal_settings.html#download-images-locally
> 
> Are you using the official docker image?

Yes, I installed according to the instructions - https://github.com/wallabag/docker

**Comment by j0k3r** [comment]
All right, the extension is missing.
